### DISCUSSION TITLE
Relationship between Call for Tender and Buyer

### DISCUSSION THREAD

**Discussion by brechtwyns** [eprocurementontology/eprocurementontology#488] [Q&A]
Should there be a direct relationship between the Call for Tender and the Buyer, given that there already is a direct relationship with Procuring Entity? How do we define that relationship?

**Comment by makxdekkers** [comment]
A Call For Tender could have a relationship with Buyer, e.g. Call For Tender 'is procured for' Buyer.

**Comment by muricna** [comment]
A Call for Tender is published for/by a Buyer it is not procured for.  What is procured is the service, works etc

**Comment by makxdekkers** [comment]
So, we could have the relationship:
Call For Tender "is published for" Buyer.
Should this be added to the model?

**Comment by muricna** [comment]
For me this is ok, but I am not sure how it fits in with procuring entity

**Comment by makxdekkers** [comment]
Call For Tender "is published by" Procuring Entity (already in the model)
Call For Tender "is published for" Buyer (could be added)
If an organisation does the procurement for itself, both Procuring Entity and Buyer would point to the same organisation.

**Comment by JachymHercher** [comment]
As @muricna mentioned in #6, it's not quite clear who publishes and what level of detail we want to go to in describing the process.

**Comment by eprocurementontology** [comment]
to be discussed when covering documents

**Comment by paulakeen** [comment]
Getting the relation of the Tenders submitted with the Buyer can be obtained through the Procedure.
